### PR TITLE
Support python 3.10

### PR DESCRIPTION
**Pull request by varac**
Python 3.10 break at different places, this PR fixes them

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,37 +1,34 @@
+# -*- coding: utf-8 -*-
```
Comment (MoritzR) [inline setup.py:1]: Are these comments needed?
Removing them seems to make no difference for more.

**Review by MoritzR — COMMENTED** [review]
Thanks for the PR, it looks good!
Just one question from my side (see the other comment).

File: setup.py (hunk 1 above)
Comment (varac) [reply setup.py]: see https://docs.python.org/3/reference/lexical_analysis.html#encoding-declarations
They are not needed, and indeed added by my pre-commit config. I took them out.

**Review by MoritzR — APPROVED** [review]

**Comment by MoritzR** [comment]
The changes are now available at version 0.8.1

**Comment by varac** [comment]
Works, thanks !

**Pull request merged**